src/Microsoft.XmlSerializer.Generator/pkg/build/Microsoft.XmlSerializer.Generator.targets

```diff
@@ -6,28 +6,40 @@
     <_SerializerCsIntermediateFolder>$(IntermediateOutputPath)$(_SerializationAssemblyName).cs</_SerializerCsIntermediateFolder>
     <_SGenWarningText>SGEN: Fail to generate the serializer for $(AssemblyName)$(TargetExt). Please follow the instructions at https://go.microsoft.com/fwlink/?linkid=858594 and try again.</_SGenWarningText>
     <_SerializationAssemblyDisabledWarnings>$(NoWarn);219;162;$(SerializationAssemblyDisabledWarnings)</_SerializationAssemblyDisabledWarnings>
+    <_SgenRspFilePath>$(IntermediateOutputPath)sgen.rsp</_SgenRspFilePath>
+    <_CscRspFilePath>$(IntermediateOutputPath)csc.rsp</_CscRspFilePath>
   </PropertyGroup>
+
   <Target Name="GenerateSerializationAssembly" AfterTargets="Build">
```

Comment:
Separate issue, but these targets are not friendly to incremental build, nor clean, nor will they actually fail the build when they fail.